include/eeprom/core/messages.hpp

```diff
@@ -29,5 +29,18 @@ struct ReadEepromMessage {
 /** The write to eeprom message */
 using WriteEepromMessage = EepromMessage;
 
+struct ConfigResponseMessage {
+    eeprom::hardware_iface::EEPromChipType chip;
+    types::address addr_bytes;
+    types::data_length mem_size;
+};
+
+using ConifgRequestCallback = void (*)(const ConfigResponseMessage&, void*);
```

Comment:
```suggestion
using ConfigRequestCallback = void (*)(const ConfigResponseMessage&, void*);
```
also maybe a `std::function`?